test/services/onlinetesting/OnlineTestServiceSpec.scala

```diff
@@ -187,11 +182,70 @@ class OnlineTestServiceSpec extends PlaySpec with BeforeAndAfterEach with Mockit
   }
 
   "register and invite application" should {
+
+    "issue one email for invites to SJQ for GIS candidates" in new OnlineTest {
+
```

Comment:
Complex tests may be a hint of having complicate code that needs to be refactored. I know sometimes it is impossible to make it smaller but it is worth to revisit this and see if we can improve it in the original service and then, these tests